client/lib/posts/actions.js

```diff
@@ -100,19 +98,20 @@ PostActions = {
 	/**
 	 * Start keeping track of edits to a new post
 	 *
-	 * @param {Number} siteId  Site ID
+	 * @param {Object} site    Site object
 	 * @param {Object} options Edit options
 	 */
-	startEditingNew: function( siteId, options ) {
+	startEditingNew: function( site, options ) {
 		var args;
 		options = options || {};
 
 		args = {
 			type: 'DRAFT_NEW_POST',
-			siteId: siteId,
+			siteId: site.ID,
 			postType: options.type || 'post',
 			title: options.title,
 			content: options.content,
+			site,
```

Comment:
Does the `site` really need to be passed as property of the `DRAFT_NEW_POST` action? It doesn't seem to be used in any of the stores that handle it. It doesn't do any harm if it's there, but is most likely redundant.

Reply:
I get it after seeing #16019 -- the `site` param is indeed needed. I'd only remove the redundant `siteId` param from the `DRAFT_NEW_POST` action.